docs/interpreter/hive.md

```diff
@@ -124,6 +124,16 @@ Hive Interpreter will be deprecated and merged into JDBC Interpreter. You can us
     <td></td>
     <td><b>( Optional ) </b>Other properties used by the driver of <code>%hive(${prefix})</code> </td>
   </tr>
+  <tr>
```

Comment:
@prabhjyotsingh Hive docs is deprecated: https://github.com/apache/zeppelin/blob/master/docs/interpreter/hive.md#important-notice

So it'll be better to add this info to https://github.com/apache/zeppelin/blob/master/docs/interpreter/jdbc.md#apache-hive instead.